picard/tagger.py

```diff
@@ -1094,6 +1213,8 @@ def process_picard_args():
                         help="location of the configuration file (starts a stand-alone instance)")
     parser.add_argument("-d", "--debug", action='store_true',
                         help="enable debug-level logging")
+    parser.add_argument("-e", "--exec", nargs="+", action='append',
+                        help="send command (arguments can be entered after space) to a running instance")
```

Comment:
@rdswift and how do you see `-e help`?

My idea is to
1) add your suggestion
2) scan `exec` args for help. If help is present, print help instead of passing the args to the pipe
3) just use `print`(?) for some help message, as consistent with `picard --help` as possible

Any better ideas or should I go this way?

Reply:
What you suggest sounds good to me.  I also think if you could include a brief note about being able to have multiple commands and them processed in the order they appear.

Reply:
Just to clarify, if a command with arguments is entered does the wole thing need to be in quotes?  For example `-e lookup path/to/my/eac/log/file` would need to be entered as `-e "lookup path/to/my/eac/log/file"`?  If so, the brief help should probably mention that as well.

Reply:
Nah, you don't need the quotes, unless the path contains spaces (which is an expected behavior imo)